models/system/databricks-dolly-v2-12b/description.md

```diff
@@ -0,0 +1,21 @@
+The dolly-v2-12b model is a large, instruction-following language model created by Databricks, Inc. and is licensed for commercial use. It is based on EleutherAI’s Pythia-12b and was trained on a 15,000-record instruction corpus generated by Databricks employees, which was released under a CC-BY-SA license. It is not a state-of-the-art model, but it does exhibit high-quality instruction-following behavior that is not characteristic of the foundation model on which it is based. The model is available in three sizes: dolly-v2-12b, dolly-v2-7b, and dolly-v2-3b.
```

Comment:
It may not be that helpful to include the three sizes since we've picked just one of them to include for this asset, unless you want to get into why a customer should choose this particular size instead of the others.